gma/integration_test/src/integration_test.cc

```diff
@@ -2818,12 +2889,12 @@ TEST_F(FirebaseGmaUmpTest, TestCanRequestAdsEEA) {
   using firebase::gma::ump::ConsentRequestParameters;
   using firebase::gma::ump::ConsentStatus;
 
-  TEST_REQUIRES_USER_INTERACTION;
```

Comment:
Why do these no longer need user interaction?

Reply:
Because I was able to add GetDebugDeviceId() to our test framework, so it can specify the debug location on hardware.